3.6/init.sh

```diff
@@ -1,21 +1,49 @@
-OSMFILE=$1
-PGDIR=$2
-THREADS=$3
+#!/bin/bash -ex
 
-rm -rf /data/$PGDIR && \
-mkdir -p /data/$PGDIR && \
+DATA_DIR=/app/src/data
+OSMFILE=${DATA_DIR}/data.osm.pbf
 
-chown postgres:postgres /data/$PGDIR && \
+if [ "$PBF_URL" = "" ]; then
+  echo "You need to specify the environmental PBF_URL"
+  echo "docker run -e PBF_URL=http://download.geofabrik.de/europe/monaco-latest.osm.pbf ..."
+  exit 1
+fi;
 
-export  PGDATA=/data/$PGDIR  && \
-sudo -u postgres /usr/lib/postgresql/12/bin/initdb -D /data/$PGDIR && \
-sudo -u postgres /usr/lib/postgresql/12/bin/pg_ctl -D /data/$PGDIR start && \
+if [ "$REPLICATION_URL" = "" ]; then
+  echo "You need to specify the environmental REPLICATION_URL"
+  echo "docker run -e REPLICATION_URL=http://download.geofabrik.de/europe/germany-updates/ ..."
+  exit 1
+else
+  sed -i "s|__REPLICATION_URL__|$REPLICATION_URL|g" /app/src/build/settings/local.php
+fi;
+
+
+if [ "$IMPORT_WIKIPEDIA" = "true" ]; then
```

Comment:
It would be great if we could also move 

https://github.com/mediagis/nominatim-docker/blob/87565e15d5a0988a876437f2073efa9253978acd/3.6/Dockerfile#L90

https://github.com/mediagis/nominatim-docker/blob/87565e15d5a0988a876437f2073efa9253978acd/3.6/Dockerfile#L97

in the init.sh like you're doing it with wikipedia. Also I think we could rename it to ```IMPORT_POSTCODES_...``` to be consistent. What do you say?

Reply:
I've just made that change.